src/main/java/net/lenni0451/spm/commands/subs/Download_Sub.java

```diff
@@ -116,6 +122,20 @@ public void getTabComplete(List<String> tabs, String[] args) {
                 tabs.add(urlPart + ".jar");
             }
         }
+        if (args.length == 2 && (args[0].equalsIgnoreCase("direct") || args[0].equalsIgnoreCase("spigot"))) {
+	        try {
```

Comment:
A try-catch block should not be needed in tab complete code. You could do something like this:
`````java
File[] files = PluginManager.getInstance().getPluginUtils().getPluginsDirectory().listFiles();
if (files != null) {
...
`````

Reply:
Implemented